packages/ckeditor5-page-break/src/pagebreakediting.ts

```diff
@@ -90,19 +91,19 @@ export default class PageBreakEditing extends Plugin {
 					const hasPageBreakAfter = element.getStyle( 'page-break-after' ) == 'always';
 
 					if ( !hasPageBreakBefore && !hasPageBreakAfter ) {
-						return;
+						return null;
 					}
 
 					// The "page break" div accepts only single child or no child at all.
 					if ( element.childCount == 1 ) {
 						const viewSpan = element.getChild( 0 );
 
 						// The child must be the "span" element that is not displayed.
-						if ( !viewSpan.is( 'element', 'span' ) || viewSpan.getStyle( 'display' ) != 'none' ) {
-							return;
+						if ( !viewSpan!.is( 'element', 'span' ) || viewSpan.getStyle( 'display' ) != 'none' ) {
+							return null;
 						}
 					} else if ( element.childCount > 1 ) {
-						return;
+						return null;
```

Comment:
I'm not sure if this is the correct way to go, to replace `undefined` with `null`. This might change the logic. But the type for method `view` expects type

```( element: Element ) => Match | null )```

so `null` is expected and not `undefined`.

Reply:
It was intentional to omit `undefined` from the callback signature. So, callbacks must be explicit that they return no match. Otherwise, `undefined` could be returned silently if not all branches have returns.